sdk/communication/communication-identity/review/communication-identity.api.md

```diff
@@ -21,28 +21,47 @@ export class CommunicationIdentityClient {
     constructor(connectionString: string, options?: CommunicationIdentityClientOptions);
     constructor(endpoint: string, credential: KeyCredential, options?: CommunicationIdentityClientOptions);
     constructor(endpoint: string, credential: TokenCredential, options?: CommunicationIdentityClientOptions);
-    createUser(options?: OperationOptions): Promise<CommunicationUserIdentifier>;
+    createUser(options?: CreateUserOptions): Promise<CommunicationUserIdentifier>;
     createUserAndToken(scopes: TokenScope[], options?: CreateUserAndTokenOptions): Promise<CommunicationUserToken>;
     deleteUser(user: CommunicationUserIdentifier, options?: OperationOptions): Promise<void>;
     getToken(user: CommunicationUserIdentifier, scopes: TokenScope[], options?: GetTokenOptions): Promise<CommunicationAccessToken>;
     getTokenForTeamsUser(options: GetTokenForTeamsUserOptions): Promise<CommunicationAccessToken>;
+    getUser(user: CommunicationUserIdentifier, options?: OperationOptions): Promise<CommunicationIdentityGetResult>;
     revokeTokens(user: CommunicationUserIdentifier, options?: OperationOptions): Promise<void>;
 }
 
 // @public
 export interface CommunicationIdentityClientOptions extends CommonClientOptions {
 }
 
+// @public
+export interface CommunicationIdentityGetResult {
```

Comment:
Can we just call this `CommunicationIdentity`? See also my other comment about using the same type for the POST with externalId.

Reply:
Renamed to CommunicationUserDetail per sdk review